File: 2_method.py

class car:
    def __init__(self, model, color):
        self.model = model
        self.color = color

    def carInfo(self):
        print(f"car model: {self.model} and car color: {self.color}")


myCar = car('BMW','Black')
myCar.carInfo()

File: 2_method.cpp
#include<bits/stdc++.h>
using namespace std;

class car
{
    public:
    string model;
    string color;

    car(string model, string color)
    {
        this->model = model;
        this->color = color;
    }
    void carInfo()
    {
        cout<<model<<" "<<color<<endl;
    }
};
int main()
{
    car myCar("Toyota", "Red");
    myCar.carInfo();
    return 0;
}